### PR TITLE
Cleanup in the phase field module

### PR DESCRIPTION
**Pull request by dschwen**
Refs #8453

### CODE REVIEW

**Review by permcody — APPROVED** [review]
Nice!

File: modules/phase_field/include/action/BicrystalBoundingBoxICAction.h [hunk 1]
```diff
@@ -11,7 +11,7 @@
 #include "Action.h"
 
 /**
- * Automatically generates all variables to model a polycrystal with op_num orderparameters
+ * Bycrystal using a bounding box
```
Comment (andrsd) [inline modules/phase_field/include/action/BicrystalBoundingBoxICAction.h]: Bicrystal (?)

Comment (permcody) [inline modules/phase_field/include/action/BicrystalBoundingBoxICAction.h]: Ha!

Comment (andrsd) [inline modules/phase_field/include/action/BicrystalBoundingBoxICAction.h]: I think I saw that in 4 other spots as well, but one never knows with these scientists ;-)

**Pull request merged**